srcpkgs/Zed/template

```diff
@@ -0,0 +1,21 @@
+# Template file for 'Zed'
+pkgname=Zed
+version=0.136.2
+revision=1
+build_style=cargo
+make_install_args="--path crates/zed"
+hostmakedepends="pkg-config protobuf"
+makedepends="alsa-lib-devel fontconfig-devel libxcb-devel libxkbcommon-devel libzstd-devel wayland-devel vulkan-loader openssl-devel libgit2-devel"
+checkdepends="xorg-fonts"
+short_desc="High-performance, multiplayer code editor"
+maintainer="SpidFightFR <spidfight@swisscows.email>"
+license="GPL-3.0-or-later, Apache-2.0, AGPL-3.0-or-later"
+homepage="https://zed.dev/"
+distfiles="https://github.com/zed-industries/zed/archive/v${version}.tar.gz"
+checksum="caf4f16fcc8490fc220a507d2287cfac6a84041c03bd9f51b5b4f63f0704737e"
+
+post_install() {
+	vlicense ${wrksrc}/LICENSE-{GPL,APACHE,AGPL}
+	vinstall ${FILESDIR}/Zed.desktop 644 usr/share/applications
+	vinstall ${FILESDIR}/Zed.png 644 usr/share/pixmaps
```

Comment:
There's the pic in the sources, i'll change that tomorrow to make this future-proof.